modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -1092,6 +1092,7 @@ const UIDropdown = UIElement.extend(/** @lends LuCI.ui.Dropdown.prototype */ {
 				'maxlength': this.options.maxlength,
 				'placeholder': this.options.custom_placeholder ?? this.options.placeholder,
 				'inputmode': 'text',
+				"enterkeyhint": "done"
```

Comment:
change to `'` to align with rest of code.

Reply:
Have been completed and pushed again, thanks for pointing that out.